server/index.js

```diff
@@ -0,0 +1,40 @@
+const express = require('express');
+const multer = require('multer');
+const bodyParser = require('body-parser');
+const db = require('../db/index');
+let app = express();
+
+app.use(express.static(__dirname + '/../client/dist'));
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({extended: true}));
+
+app.get('/rooms/:roomid/similarListings', (req, res) => {
+  //
+  let roomId = req.body;
```

Comment:
Are the tests passing for this change? A more standard way for getting the parameters is using req.params.roomid